playbooks/playbook.yaml

```diff
@@ -42,37 +44,37 @@
         - V4_CFG_MANAGE_STORAGE|bool
       tags:
         - viya
-    - name: baseline role install
+    - name: baseline role install # noqa: name[casing]
```

Comment:
Ok, I understand the addition of the comment here. Does this show up in the task as it's run?

Reply:
Ran through a baseline install, the comment does not appear with the task name in the console output.

```
TASK [baseline role install] ***********************************************************************************************
Monday 07 August 2023  14:45:23 +0000 (0:00:00.075)       0:00:04.899 *********

```